python/tank/authentication/login_dialog.py

```diff
@@ -142,6 +188,12 @@ def _link_activated(self, site):
                 self.ui.message, "Can't open '%s'." % forgot_password
             )
 
+    def _toggle_sso(self):
+        self._use_sso = not self._use_sso
+        self.ui.login.setEnabled(not self._use_sso)
+        self.ui.password.setEnabled(not self._use_sso)
+        # self.ui.message.setVisible(not self._use_sso)
```

Comment:
This file is under construction, so I did not clean it all up nicely yet. Yet being the operative word.